examples/simple.rs

```diff
@@ -34,11 +38,44 @@ async fn main() -> Result<(), Report> {
 
     // Download the file(s).
     let reqwest_rs = "https://github.com/seanmonstar/reqwest/archive/refs/tags/v0.11.9.zip";
-    let downloads = vec![Download::try_from(reqwest_rs).unwrap()];
+    let fake = format!("{}.fake", &reqwest_rs);
+    let downloads = vec![
+        Download::try_from(reqwest_rs).unwrap(),
+        Download::try_from(fake.as_str()).unwrap(),
+    ];
     let downloader = DownloaderBuilder::new()
         .directory(PathBuf::from("output"))
         .build();
-    downloader.download(&downloads).await;
+    let summaries = downloader.download(&downloads).await;
+
+    // Display results.
+    display_summary(&summaries);
 
     Ok(())
 }
+
+fn display_summary(summaries: &[Summary]) {
+    let mut table = Table::new();
+    let header = Row::from(vec!["File", "Size", "Status", "Error"]);
+    table.set_header(header);
+    summaries.iter().for_each(|s| {
+        let mut error = String::new();
+        let status = match s.status() {
+            Status::Success => String::from("✅"),
+            Status::Fail(s) => {
+                error = s.to_string();
+                error.truncate(50);
+                error.push_str("...");
```

Comment:
We should check to see if the string is longer than 50, adding `...` only if true

Reply:
✅